generator/src/main/java/com/linkedin/pegasus/generator/TemplateSpecGenerator.java

```diff
@@ -602,7 +602,6 @@ private MapTemplateSpec generateMap(MapDataSchema schema, ClassTemplateSpec encl
 
     final MapTemplateSpec mapClass = (MapTemplateSpec) classInfo.definedClass;
     registerClassTemplateSpec(schema, mapClass);
-
```

Comment:
nit: Typically if there are no changes in a file, i would not do any whitespace fixes.

Reply:
Thanks, I will remove this